tests/scripts/task_python_frontend.sh

```diff
@@ -53,3 +53,7 @@ python3 -m pytest -v tests/python/frontend/caffe2
 
 echo "Running relay DarkNet frontend test..."
 python3 -m pytest -v tests/python/frontend/darknet
+
+
+echo "Running relay to_onnx test..."
+python3 -m pytest -v tests/python/frontend/to_onnx
```

Comment:
I think this should be in another script cause it's not really a frontend.

Reply:
moved it to integration.sh